pyproject.toml

```diff
@@ -21,8 +21,12 @@ include = [
 [tool.poetry.dependencies]
 python = "^3.6"
 dataclasses = {version = ">=0.7 <0.9", python = "3.6"}
+docstring_parser = {version = "^0.7.3", optional = true}
 typing-extensions = {version = "^3.7.4.3", python = "3.6 || 3.7"}
 
+[tool.poetry.extras]
+docstring-parser = ["docstring_parser"]
```

Comment:
This should really be named `numpy` or `numpy-style`.
`docstring-parser` is too generic and it would be confusing because we don't need it for other docstrings style.